constants/competition.ts

```diff
@@ -4,6 +4,6 @@
  */
 export const COMPETITION_DATES = {
 	// note: month starts at 0, not 1.
-	START_DATE: new Date(2022, 7, 18),
-	END_DATE: new Date(2022, 7, 22),
+	START_DATE: new Date(2022, 7, 16),
```

Comment:
Note that it's different from the description `Sets competition to start on August 16th, 00:00 and ending on August 23rd, 00:00`

Reply:
Got it. It's `monthIndex`